CONTRIBUTING.md

```diff
@@ -151,15 +151,15 @@ make docker
 
 #### <a name="adapter">Writing a Meshery Adapter</a>
 
-Meshery uses adapters to provision and interact with different service meshes. Follow these instructions to create a new adapter or modify and existing adapter.
+Meshery uses adapters to provision and interact with different service meshes. Follow these instructions to create a new adapter or modify an existing adapter.
 
 1. Get the proto buf spec file from Meshery repo:
    `wget https://raw.githubusercontent.com/meshery/meshery/master/server/meshes/meshops.proto`
 1. Generate code
    1. Using Go as an example, do the following:
       - adding GOPATH to PATH: `export PATH=$PATH:$GOPATH/bin`
       - install grpc: `go get -u google.golang.org/grpc`
-      - install protoc plugin for go: `go get -u github.com/golang/protobuf/protoc-gen-go`
+      - install protoc plugin for go: `go get -u google.golang.org/protobuf`
       - Generate Go code: `protoc -I meshes/ meshes/meshops.proto --go_out=plugins=grpc:./meshes/`
```

Comment:
Maybe this can be changed as....


```suggestion
       - Generate Go code: `protoc -I server/meshes/ server/meshes/meshops.proto --go_out=plugins=grpc:./server/meshes/`
```

Could you check if this works? @abdullah1308